params-2025x.json

```diff
@@ -0,0 +1,1468 @@
+                },
+                "anisotropyRotation": {
+                    "name": "Anisotropy Rotation",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 4,
+                    "group": "Base"
+                },
+                "normal": {
+                    "name": "Normal",
+                    "type": "normal",
+                    "default": [0, 0, 1],
+                    "uiOrder": 5,
+                    "group": "Base"
+                },
+                "displacement": {
+                    "name": "Displacement",
+                    "type": "vector",
+                    "default": [0, 0, 0],
+                    "uiOrder": 6,
+                    "group": "Base"
+                },
+                "translucency": {
+                    "name": "Translucency",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 7,
+                    "group": "Base"
+                },
+                "translucencyColor": {
+                    "name": "Translucency Color",
+                    "type": "color",
+                    "max": [1, 1, 1],
+                    "default": {
+                        "$ref": "#/materials/generic/parameters/albedo/default"
+                    },
+                    "uiOrder": 8,
+                    "group": "Base"
+                },
+                "transparency": {
+                    "name": "Transparency",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 9,
+                    "group": "Base"
+                },
+                "cutoutOpacity": {
+                    "name": "Cut-Out Opacity",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 1,
+                    "uiOrder": 10,
+                    "group": "Base"
+                },
+                "specular": {
+                    "name": "Specular",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 1,
+                    "uiOrder": 11,
+                    "group": "Base"
+                },
+                "specularTint": {
+                    "name": "Specular Tint",
+                    "type": "color",
+                    "max": [1, 1, 1],
+                    "default": [1, 1, 1],
+                    "uiOrder": 12,
+                    "group": "Base"
+                },
+                "thinFilm": {
+                    "name": "Thin Film",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 35,
```

Comment:
`uiOrder` needs to be adjusted too. This is what GUIs are supposed to take into account. Reason: [JSON objects are unordered.](https://datatracker.ietf.org/doc/html/rfc8259#section-1)